src/Illuminate/Database/Connectors/MySqlConnector.php

```diff
@@ -24,19 +24,10 @@ public function connect(array $config)
         $connection = $this->createConnection($dsn, $config, $options);
 
         if (! empty($config['database'])) {
-            $connection->exec("use `{$config['database']}`;");
+            $connection->exec("USE `{$config['database']}`;");
```

Comment:
I think we use lowercase instead of uppercase for operations.

Reply:
I'm not sure, but I think this is a bug fix for when the server defaults to case sensitive collation, before we get to set it to insensitive. MySQL commands are defined in uppercase.

Reply:
To my knowledge, keywords in SQL language are always case-insensitive. Could you please share a MySQL server setting that makes it case-sensitive?